disdrodb/data_transfer/scripts/download_disdrodb_archive.py

```diff
@@ -0,0 +1,72 @@
+# -----------------------------------------------------------------------------.
+# Copyright (c) 2021-2022 DISDRODB developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------.
+####################################################################
+## Wrapper to download disdrodb archives by command lines ##
+####################################################################
+import click
+
+
+@click.command()
+@click.option(
+    "--disdrodb_dir",
+    required=True,
+    help="Raw data folder path (eg : /home/user/DISDRODB/Raw). Is compulsory.",
+)
+@click.option(
+    "--data_sources",
+    multiple=True,

```

Comment:
I would add to this options the :   
```
    type=str,
    show_default=True,
    default="",
```
See:  https://github.com/ltelab/disdrodb/blob/main/disdrodb/l0/l0_processing.py#L1179.

Otherwise a station like '10' became a number instead of string ;)

Reply:
ok